src/pudl/metadata/templates/codemetadata.rst.jinja

```diff
@@ -0,0 +1,29 @@
+{% if is_header %}
+=====================================================================================================
+PUDL Code Metadata
+=====================================================================================================
+*The following tables map from codes used in raw data to labels used in the PUDL database.*
+*Also included is a table of non-standard codes that are mapped to canonical, standardized codes.*
+{% endif %}
+
+| **{{ Encoder.name }}**
```

Comment:
For easier navigation / linking & consistency can we make each coding table a header like in the [PUDL DB Data DIctionary](https://catalystcoop-pudl.readthedocs.io/en/latest/data_dictionaries/pudl_db.html). I think the larger headers will help distinguish between the different coding tables too, since they have multiple sections (description, codes, fixes) which run together a bit now.